conf/modules.config

```diff
@@ -105,8 +157,9 @@ process {
         memory = { 8.GB * task.attempt }
         time = { 15.m * task.attempt }
         publishDir = [
-            path: { "${params.outdir}/HAPPY/vcf_giab" },
+            path: { "${params.outdir}/HAPPY/vcf_giab/" },
```

Comment:
a "/" is added, is this an issue?

Reply:
No, but did remove slash in publish path(s) for consistency